### PR TITLE
Configure password staleness

### PR DESCRIPTION
**Pull request by malteo**

### CODE REVIEW

File: cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/EnvironmentDTO.java [hunk 1]
```diff
@@ -344,6 +345,11 @@ public class EnvironmentDTO implements DTO
     @Valid
     private MailCustomization billingBudgetThresholdCrossed = new MailCustomization();
 
+    @ApiModelProperty( "Sent to the customer before password expiral" )
```
Comment (manuelmazzuola) [inline cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/EnvironmentDTO.java:348]: expires?
mi sa expiral non esiste

File: cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/EnvironmentDTO.java [hunk 2]
```diff
@@ -2094,6 +2110,15 @@ public static class ConfigurationEnvironment implements DTO
         @NotNull
         private Long maxFileSize = 0L;
 
+        @ApiModelProperty( "Maximum number of days for a password to be stale, a null or <1 value disables expiral" )
```
Comment (manuelmazzuola) [inline cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/EnvironmentDTO.java:2113]: ⬆️

File: cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/MailEnvironmentDTO.java [hunk 3]
```diff
@@ -347,6 +347,13 @@ public void setTranslations( Translations translations )
         @Valid
         private MailConfiguration billingBudgetThresholdCrossed;
 
+        /**
+         * Sent to the customer before password expiral
```
Comment (manuelmazzuola) [inline cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/MailEnvironmentDTO.java:351]: ⬆️

**Review by manuelmazzuola — CHANGES_REQUESTED** [review]

File: cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/EnvironmentDTO.java (hunk 1 above)
Comment (malteo) [reply cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/EnvironmentDTO.java:348]: LOL è

> expiry

https://www.collinsdictionary.com/dictionary/english/expiry
https://dictionary.cambridge.org/dictionary/english/expiry

**Pull request merged**